addons/mail/static/src/components/follower_list_menu/follower_list_menu.xml

```diff
@@ -16,7 +18,7 @@
                                 Add Followers
                             </a>
                         </t>
-                        <t t-if="thread.followers.length > 0">
+                        <t t-if="thread.followers and thread.followers and thread.followers.length > 0">
```

Comment:
`thread.followers` twice.
Also what's the deal with checking truthy `thread.followers`? It's a relational field, so it's value is a collection/set and it's always truthy. What's going on if that's not the case?